Gemfile

```diff
@@ -6,6 +6,14 @@ ruby "3.2.2"
 # Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
 gem "rails", "~> 7.0.6"
 
+gem "devise"
+
```

Comment:
Please remove extra lines between gems.
Install rubocop and fix all the offenses.

Reply:
Sure, I will fix all the offense using robocop